src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -135,38 +136,39 @@ static void save_memory_to_file(char* addr, size_t size) {
 
 // Shared Memory Implementation Details
 
-// Note: the solaris and linux shared memory implementation uses the mmap
+// Note: the linux shared memory implementation uses the mmap
 // interface with a backing store file to implement named shared memory.
 // Using the file system as the name space for shared memory allows a
 // common name space to be supported across a variety of platforms. It
 // also provides a name space that Java applications can deal with through
 // simple file apis.
 //
-// The solaris and linux implementations store the backing store file in
+// The linux implementation stores the backing store file in
```

Comment:
Do we need this in a shared file? What do other platforms do?

Reply:
This comment is incorrect for Linux because of containers. So, I deleted the comment.